gammapy/estimators/flux.py

```diff
@@ -68,12 +85,23 @@ def __init__(
         selection_optional=None,
         fit=None,
         reoptimize=False,
+        norm=None,
     ):
-        self.norm_values = norm_values
-        self.norm_min = norm_min
-        self.norm_max = norm_max
-        self.norm_n_values = norm_n_values
+
         self.source = source
+        if norm is None:
+            norm = Parameter(
```

Comment:
To further clarify that the parameter is not a parameter in the model sense, but more a configuration class we could also allow users to pass a dict as well and create the parameter internally.  Support both.

Reply:
done